htmresearch/algorithms/column_pooler.py

```diff
@@ -174,16 +192,127 @@ def compute(self, feedforwardInput=(), lateralInputs=(),
 
     @param  learn (bool)
             If True, we are learning a new object
-    """
 
+    @param predictedInput (sequence)
+           Sorted indices of active feedforward input bits
```

Comment:
This parameter needs a good docstring. This made me think these were "predicted active cells", when really these cells aren't necessarily active (and this is vital to how the algorithm works.)